cli/internal/packager/deploy.go

```diff
@@ -17,19 +22,21 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-func Deploy(packageName string, confirm bool, componentRequest string) {
+var validExtensions = []string{".tar.zst", ".tar", ".zip"}
```

Comment:
things like this I prefer to makes constants at the top of the file or move to the config package.

Reply:
I'll move it to the config package, you're right that it makes more sense there.

Golang doesn't allow making slices/arrays constants because arrays are essentially pointers to their elements, the best we can do is make the array a fixed size. 
https://stackoverflow.com/a/13137568

 Given your other comment on checking filetype in the future

Reply:
I added a initialization function to give us the immutability similarity as described here https://qvault.io/golang/golang-constant-maps-slices/